test/MSVC/msvc.py

```diff
@@ -57,50 +59,6 @@
 env.Object('slow', 'foo.cpp', PCH=0)
 """)
 
-test.write('test.cpp', '''
```

Comment:
I made a fixture for the new test, so the msvc test was just update to share that fixture.